subprojects/docs/src/snippets/tasks/incrementalTask/groovy/build.gradle

```diff
@@ -9,6 +10,7 @@ task originalInputs() {
 
 // tag::updated-inputs[]
 task updateInputs() {
+    outputs.dir('inputs')
```

Comment:
Do you mean `inputs.dir('inputs')`?

Reply:
No. The outputs of this task is the directory `inputs`. It changes files in there.